Makefile

```diff
@@ -0,0 +1,16 @@
+CPPFLAGS:=-I./lua/src
+CFLAGS:=--std=c99 -g -Wall -O2 -fPIC -shared
+
+.PHONY: all lua clean
+
+all: lua runtime/tcore.o
```

Comment:
I think we should add another phony called runtime (or titanruntime) and add `runtime/tcore.o` as a dependency of it.